rfcs/text/0040-volume-device.md

```diff
@@ -279,9 +360,10 @@ Stage 3: Document resolutions for all existing concerns. Any new concerns should
 
 The following are the people that consulted on the contents of this RFC.
 
- * @Trinity2019    | author
- * @ricardoungureanu| reviewer
- * @stanek-michal  | reviewer
+ * @Trinity2019      | author
```

Comment:
```suggestion
 * @Trinity2019      | author, sponsor
```

I overlooked we didn't have a sponsor listed yet. @Trinity2019 would you be willing to list yourself as sponsor?

The sponsor is someone at Elastic who ([source](https://github.com/elastic/ecs/blob/main/rfcs/PROCESS.md#responsibilities-in-this-process)):

* can be the same person as the contributor if they're someone at Elastic that can take ownership of this change
* is involved at least from stage 1 onward if a different person than the contributor
signs off on each stage if a different person than the contributor
* takes or coordinates ownership of the addition in terms of support and maintenance after the RFC process is completed

Reply:
yes, I can be the sponsor. Thanks @ebeahan for reviews :)